internal/exec/engine.go

```diff
@@ -103,6 +110,41 @@ func (e Engine) Run(stageName string) error {
 	return nil
 }
 
+// isConfigCacheExist checks if we already have a cached config.
+// TODO(sohan): This is a hack to emit the journald message only
+// once for the base config (https://github.com/coreos/ignition/issues/950)
+func (e *Engine) isConfigCacheExist(cache string) error {
+	var haveConfig bool
+	if _, err := os.Stat(e.ConfigCache); err != nil {
+		if !os.IsNotExist(err) {
+			return err
+		}
+		haveConfig = true
+	}
+	if !haveConfig {
+		if err := logStructuredJournalEntry("base", "system", "false"); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// logStructuredJournalEntry logs infomration related to
+// a user/base config into the systemd journal log.
+func logStructuredJournalEntry(config string, src string, referenced string) error {
```

Comment:
`referenced` should be a `bool`

Reply:
You're right, but the `Journal.Send()` function has a map parameter which takes only string values.
https://github.com/coreos/ignition/pull/958/files#diff-e3491b7c0e160811ce6761d5a1117558R135

Reply:
I can use `strconv.FormatBool(v)` to convert a boolean value to a string.

Reply:
Right, when serializing you lose types, but in general until that last moment we should try to stay in the realm of strong typing. :)